edx_api/email_settings.py

```diff
@@ -0,0 +1,60 @@
+"""
+API client for interacting with change email settings
+"""
+import logging
+from urllib import parse
+
+log = logging.getLogger(__name__)
+
+
+class ChangeEmailSettings(object):
+    """
+    API client for interacting with change email settings
+    """
+    api_url = "/api/change_email_settings"
+
+    def __init__(self, requester, base_url):
+        self.requester = requester
+        self.base_url = base_url
+
+    def change_settings(self, payload):
+        """
+        Execute the client request to edX endpoint
+        Args:
+            payload (dict): request payload
+        Returns:
+            JSON response (dict)
+        """
+        response = self.requester.post(
+            parse.urljoin(self.base_url, self.api_url),
+            json=payload
+        )
+
+        try:
+            response.raise_for_status()
+        except Exception:
+            log.error(response.json())
+        return response.json()["success"]
+
+    def subscribe(self, course_id):
+        """
+        subcribe the user to receive all course emails
```

Comment:
subcribe the user to receive all course emails

```suggestion
Subscribe the user to receive all course emails```